tests/test_post_gen_project.py

```diff
@@ -0,0 +1,185 @@
+import os
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+import pytest
+from conftest import HOME
+
+sys.path.append(str(Path(__file__).resolve().parents[1]))
+from hooks.post_gen_project import (  # noqa: E402
+    copy_journal_template_files,
+    get_cookiecutter_dir,
+)
+
+
+@pytest.mark.parametrize(
+    "cookiecutters",
+    [
+        # test get_cookiecutter_dir when "scikit-package-manuscript"
+        #   cookiecutter exists.
+        # C1: with only "scikit-package-manuscript" cookiecutter.
+        ("scikit-package-manuscript",),
+        # C2: with "scikit-package-manuscript" and other cookiecutters.
+        ("scikit-package-manuscript", "foo", "bar"),
+    ],
+)
+def test_get_cookiecutter_dir(user_filesystem, cookiecutters):
+    home_path = user_filesystem
+    skm_path = home_path / ".cookiecutters" / "scikit-package-manuscript"
```

Comment:
just `expected = home_path / ".cookiecutters" / "scikit-package-manuscript"` and remove the line below.

Though I would move this down to line 35 to make things more readable.